content/en/docs/best-practices/channel-naming.md

```diff
@@ -175,9 +200,80 @@ candidate/fast/stable to denote the maturity level of the operator.  Using all
 
 Channel promotion is the notion of moving an operator from one channel to 
 another.  For example, consider the case where you have an operator version 
-1.0.1 which is found in a candidate channel, then you might decide to offer 
+`1.0.1` which is found in a candidate channel, then you might decide to offer 
 support for that version and want to move it to a stable channel.  
 
 Today, channel promotion is achieved by creating a new operator version 
-(1.0.2) that is labeled with the channel(s) you want to promote to (as well 
+(`1.0.2`) that is labeled with the channel(s) you want to promote to (as well 
 as any channels you want to keep it in).
+
+# UPGRADES 
+
+OLM provides a variety of ways to specify updates between operator versions. Before
+continue in the following recommendations, make sure you understand its options by checking [Creating an update graph with OLM][upgrade-graph].
+
+## Recommended upgrade path for patch release (bug fixes)
+
+Within a channel each patch release should be directly upgradable to the `head channel` from the latest version shipped 
+in the minor channel behind. Use skips or skipRange to provide this behaviour. (i.e. using the skipRange in the 
+`stable-3.6` patch for `3.5.<latest>: >= 3.5.<first z of the minor channel>. < 3.6.<new pacth release>`)
```

Comment:
```suggestion
`stable-3.6` patch for `3.5.<latest>: >= 3.5.<first z of the minor channel>. < 3.6.<new patch release>`)
```

aside from the typo, i can't understand what this is saying.

should it be this instead:
`stable-3.6` patch for `3.6.<latest>` would mean setting the skipRange to be `>= 3.5.<first z of the minor channel>. < 3.6.<new patch release>`, where 3.5.z represents the oldest 3.5.z version for which you can provide direct upgrade support to 3.6.latest)